packages/webapp/pages/squads/discover/featured.tsx

```diff
@@ -8,15 +8,7 @@ import InfiniteScrolling, {
 import { InfiniteData, useInfiniteQuery } from '@tanstack/react-query';
 import { ClientError } from 'graphql-request';
 import { useAuthContext } from '@dailydotdev/shared/src/contexts/AuthContext';
-import {
```

Comment:
There have been some general changes on this file, but it is for easier maintenance. Kindly check the latest ones and let me know if you need help with merging.

Reply:
Updated ✅